elastic/kibana
# [Serverless][Index Management] Hide Storage size column in Data streams

Fixes https://github.com/elastic/kibana/issues/167654

## Summary

This PR removes the Storage size column in Data stream in serverless as the data stream stats API on serverless doesn't currently return the storage size.

### How to test:

Verify that the column is not displayed in serverless:

1. Start Es with `yarn es serverless` and Kibana with `yarn serverless-{es/oblt/security}`
2. Go to Stack Management -> Index Management -> Data Streams
3. Switch on the "Include stats" toggle
4. Verify that the "Storage size" column is not shown in the table.

Verify that the column is displayed in stateful:

1. Start Es with `yarn es snapshot` and Kibana with `yarn start`
2. Go to Stack Management -> Index Management -> Data Streams
3. Switch on the "Include stats" toggle
4. Verify that the "Storage size" column is shown in the table.
